tests/unit/test_koji_wrapper.py

```diff
@@ -62,6 +62,20 @@ def test_init_with_koji_wrapper():
     assert test_wrapper.session is my_koji_wrapper.session
 
 
+def test_init_with_koji_wrapper_only():
+    """
+    GIVEN we have a valid KojiWrapper with a session,
+    WHEN a KojiWrapper object is passed as the session to a new KojiWrapper only
+    THEN this object's session is set to the session and url/topurl of the passed object
+    """
+    my_koji_wrapper = build_wrapper(None)
```

Comment:
To properly test the new behavior, we probably also need to change build_wrapper to take url and topurl params so we can start of with them empty for the test here